trino/trino.go

```diff
@@ -1881,6 +1887,17 @@ func scanNullTime(v interface{}) (NullTime, error) {
 	if len(vparts) > 1 && !unicode.IsDigit(rune(vparts[len(vparts)-1][0])) {
 		return parseNullTimeWithLocation(vv)
 	}
+	// Time literals may not have spaces before the timezone.
+	vparts = strings.Split(vv, "+")
```

Comment:
Would checking if `vv` contains a `+` be enough (using `strings.Contains()`)? It would help avoid allocating the `vparts` slice that's not used otherwise.

Reply:
Good idea, done.

Reply:
Well, it's more complicated because we have hyphens in the date part, so a simple contains is not sufficient. I now use Count we should reduce the allocations. Thanks!